src/apiwrappers/auth.py

```diff
@@ -31,3 +31,18 @@ def __repr__(self) -> str:
 
     def __call__(self) -> Dict[str, str]:
         return {"Authorization": str(self)}
+
+
+class ApiKeyAuth:
+    def __init__(self, key: str, header: str = "apikey"):
+        self.key = key
+        self.header = header
+
+    def __str__(self) -> str:
+        return f"{self.header} {self.key}"
```

Comment:
Both `BasicAuth` and `TokenAuth` do not contain a header name in `__str__` method. Could you please   provide same behaviour here or at least separate header name and header value with ':'

Reply:
modified as requested